slang-checks/src/main/java/com/sonarsource/slang/checks/CommentedCodeCheck.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * SonarSource SLang
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonarsource.slang.checks;
+
+import com.sonarsource.slang.api.ASTConverter;
+import com.sonarsource.slang.api.Comment;
+import com.sonarsource.slang.api.TopLevelTree;
+import com.sonarsource.slang.checks.api.InitContext;
+import com.sonarsource.slang.checks.api.SlangCheck;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.sonar.check.Rule;
+
+@Rule(key = "S125")
+public class CommentedCodeCheck implements SlangCheck {
+  private ASTConverter commentAnalyser;
+
+  public CommentedCodeCheck(ASTConverter astConverter) {
+    this.commentAnalyser = astConverter;
+  }
+  private static final String MESSAGE = "Remove this commented out code.";
+  @Override
+  public void initialize(InitContext init) {
+    init.register(TopLevelTree.class, (ctx, tree) -> {
+      List<List<Comment>> groupedComments =
+        groupComments(tree.allComments());
+      groupedComments.forEach(comments -> {
+        String content = comments.stream()
+          .map(Comment::contentText)
+          .collect(Collectors.joining(""));
+        if (commentAnalyser.parse(content) != null) {
+         ctx.reportIssue(comments.get(0).textRange(), MESSAGE);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this line start at column 11. ([squid:IndentationCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.slang%3Aslang&pullRequest=127&issues=AWRbS0Gr-1T-AjkxlrYC&open=AWRbS0Gr-1T-AjkxlrYC)

[ISSUE_KEY]: AWRbS0Gr-1T-AjkxlrYC